docs/products/clickhouse/concepts/plans-pricing.rst

```diff
@@ -0,0 +1,54 @@
+Aiven for ClickHouse® service plans
```

Comment:
This information is quite generic. Would it be better to have a common article for all the services in Platform? There is nothing specific for ClickHouse, except of the link to https://aiven.io/pricing?tab=plan-pricing&product=clickhouse

Reply:
@anelook , although it may seem that all services have similar sets of available plans, there are differences, e.g. between what plans ClickHouse, Apache Kafka, or  PostgreSQL offer.